tests/functional/helper.js

```diff
@@ -243,36 +243,45 @@ module.exports = {
 
   // TODO - The expected staging urls are hardcoded, we may want to
   // move these hardcoded urls out if testing in alternate environments.
-  async authenticate(clickOnPostAuthenticate = false, acceptTelemetry = false) {
+  async authenticateInBrowser(
+      clickOnPostAuthenticate = false, acceptTelemetry = false) {
+console.log("A");
```

Comment:
Please remove these debugging `console.log()` messages.

Reply:
Yes I was debugging something to fix the in-browser auth. We have an issue in the iOS auth flow.